scripts.js

```diff
@@ -0,0 +1,88 @@
+function signup() {
+  var handle = document.getElementById("handle")
+  var password = document.getElementById("password")
+
+  var url = 'https://chitter-backend-api.herokuapp.com/users';
+  var data = {user: {handle: handle.value, password: password.value}};
+
+  fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers:{
+      'Content-Type': 'application/json'
+    }
+  })
+}
+
+function userLogIn() {
+  var handleSignin = document.getElementById("handleSignIn")
+  var passwordSignin = document.getElementById("passwordSignIn")
+
+  var urlSession = 'https://chitter-backend-api.herokuapp.com/sessions';
+  var dataSession = {session: {handle: handleSignin.value, password: passwordSignin.value}};
+
+  $.ajax({
+    url: urlSession,
+    type: 'post',
+    dataType: 'json',
+    contentType: 'application/json',
+    success: function (data) {
+        var logInDetails = data;
+        loadPeeps(logInDetails);
+    },
+    data: JSON.stringify(dataSession)
```

Comment:
I didn't use this JSON.stringify thingy but it still seemed to work. Perhaps I just got lucky? : S